src/main/rx/rx_unified_dsm.c

```diff
@@ -37,26 +36,52 @@ extern uint8_t ready_for_next_telemetry;
 #define USART usart_port_defs[serial_rx_port]
 #define SPECTRUM_BIND_PIN usart_port_defs[profile.serial.rx].rx_pin
 
-void rx_serial_process_dsmx() {
+static uint8_t dsm_protocol = 0;
+#define DSMX_11_2048 0xb2
+#define DSMX_22_2048 0xa2
+#define DSM2_11_2048 0x12
+#define DSM2_22_1024 0x01
```

Comment:
lets convert these to an enum and move that to the top of the file, after the includes.
`dms_protocol` should use this enum type.